sycl/source/accessor.cpp

```diff
@@ -13,8 +13,13 @@
 namespace sycl {
 inline namespace _V1 {
 namespace detail {
-device getDeviceFromHandler(handler &CommandGroupHandlerRef) {
-  return CommandGroupHandlerRef.MQueue->get_device();
+device getDeviceFromHandler(handler &cgh) {
```

Comment:
```suggestion
device getDeviceFromHandler(const handler &cgh) {
```

Reply:
This function is declared as `__SYCL_EXPORT`, so I believe this would cause an ABI break